TypeInjections/Tool/Tool.fs

```diff
@@ -21,31 +21,38 @@ module Tool =
         let commitRepo = newRepo.Lookup<Commit>(hash)
         Commands.Checkout(newRepo, commitRepo) |> ignore
 
-    // based off command line flags, create and provide path for input/output folders
-    let interpretFlag (argvList: string list) =
-        let flag = argvList.Item(0)
-
-        // -p [inputPath] [inputPath] [outputPath]
-        if flag.Contains("-p") then
-            Console.WriteLine("***** local project comparison")
-
-            if argvList.Length <> 4 then
-                failwith "usage: program -p OLD NEW OUTPUTFOLDER"
 
-        // -l [git commit hash] [git commit hash] [OutputPath]
-        elif flag.Contains("-l") then
-            Console.WriteLine("***** git commit")
+    type options =
+        { [<Option('d', "delete", Required = false, HelpText = "use option to remove New and Old folders")>]
```

Comment:
Are `New` and `Old` the folder names? Then make it `the New and Old folders`. If not please lower case them.

Does this flag really just delete them? I don't think we should offer such functionality. Or does it delete existing ones before writing new ones? This is often called `force` (maybe this would be a more fitting name).
If it does more than just delete them, please reflect that in the help text.

Reply:
Yeah sorry for the confusion, `-d` is when the user wants to use typeCart on two git versions. When comparing git versions, the tool clones two repositories at a given `old commit` and `new commit` and then runs typeCart. I couldn't figure out whether a user would want to keep these checked out new and old repositories, so I gave them an option to keep these typeCart generated cloned repositories or delete them 

so here is the workflow of the tool when comparing git versions:
clone old project -> clone new project -> run typeCart -> (optional) delete cloned projects

If this is useless, let me know. I can remove this feature!